mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
         - 'Consuming RSS Feeds': consuming-rss.md
         - 'Consuming Atom Feeds': consuming-atom.md
         - 'Consuming Atom Entries': consuming-atom-entry.md
+        - 'Consuming Podcast Feeds': consuming-podcasts.md
```

Comment:
I agree with you and this is how I originally wanted to handle it but since there's a difference between reader and writer extensions I figured this could add confusion. Maybe we can add an Extensions header under both Reader and Writer?

Reply:
> Maybe we can add an Extensions header under both Reader and Writer?

Please use a separate page as suggested. The topic "Extensions" must be covered in the introduction but this is another rework and does not belong to this pull request.